pegasus/cache/i18n/en-US.yml

```diff
@@ -751,7 +751,6 @@
   dashboard_students_male: "Male"
   dashboard_students_female: "Female"
   dashboard_students_completed_levels: "Completed Levels"
-  dashboard_students_total_lines: "Lines of Code"
```

Comment:
Did you remove this in the pegasus gsheet as well? I believe this will just get added back as part of the staging scoop if not